packages/skill-lesson/path-to-purchase/pricing-check-context.tsx

```diff
@@ -1,15 +1,21 @@
 import * as React from 'react'
 import {Dispatch, SetStateAction} from 'react'
 import type {FormattedPrice} from '@skillrecordings/commerce-server/dist/@types'
+import type {MerchantCoupon} from '@skillrecordings/database'
+
+type MinimalMerchantCoupon = Omit<
+  MerchantCoupon & {
+    country?: string
+  },
+  'identifier'
+>
```

Comment:
Make the `usePriceCheck` types for `merchantCoupon` and `setMerchantCoupon` more compatible.